activesupport/lib/active_support/testing/time_helpers.rb

```diff
@@ -169,10 +169,21 @@ def travel_to(date_or_time, with_usec: false)
           now = date_or_time.to_time.change(usec: 0)
         end
 
-        stubbed_time = Time.now if simple_stubs.stubbing(Time, :now)
-        simple_stubs.stub_object(Time, :now) { at(now.to_f) }
-        simple_stubs.stub_object(Date, :today) { jd(now.to_date.jd) }
-        simple_stubs.stub_object(DateTime, :now) { jd(now.to_date.jd, now.hour, now.min, now.sec, Rational(now.utc_offset, 86400)) }
+        stubs = simple_stubs
+        stubbed_time = Time.now if stubs.stubbing(Time, :now)
+        stubs.stub_object(Time, :now) { at(now.to_f) }
+
+        stubs.stub_object(Time, :new) do |*args|
```

Comment:
Hum, how does `stub_object` deal with keyword args? Because `Time.new` does accept some: https://docs.ruby-lang.org/en/master/Time.html#method-c-new

Reply:
Well, if there are arguments we forward them to the original method right? If so we need to make sure keyword args are passed correctly. Hence why I suggest adding a test for that.